app/workers/evss/disability_compensation_form/submit_form526.rb

```diff
@@ -15,7 +15,7 @@ class SubmitForm526
 
       # This callback cannot be tested due to the limitations of `Sidekiq::Testing.fake!`
       sidekiq_retries_exhausted do |msg, _ex|
-        transaction_class.update_transaction(jid, :exhausted)
+        transaction_class.update_transaction(msg['jid'], :exhausted)
```

Comment:
@SMLuthi This is merged but just had a though if the `jid` wasn't available I'd think the `transaction_class` method wouldn't be either?